db/migrate/20230418075940_assing_fqdn_to_host_name.rb

```diff
@@ -0,0 +1,9 @@
+class AssingFqdnToHostName < ActiveRecord::Migration[6.1]
+  def change
+    Host.all.each do |host|
+      unless host.name.ends_with? host.domain.name
+        host.update(name: host.fqdn)
+      end
+    end
+  end
+end
```

Comment:
This migration did not work as `host.domain` was `nil`. 

Isn't hostname always in the format of FQDN? And we just display it differently according to the setting?

In my opinion, we should not change the hostname in DB.

Reply:
I see, fixed.

>  And we just display it differently according to the setting?

Yes, this is what I am trying to do with this PR.

> Isn't the hostname always in the format of FQDN?

This is currently not the case, but it will be after this is merged. Some users have shortnames stored in hostname which can cause inconsistencies.